docs/env.md

```diff
@@ -1,117 +1,260 @@
 # Environment variables
 
-We try to follow [12 factors app](https://12factor.net/) and so use
-environment variables for configuration.
+`marsha` contains several projects, two of which are configured through environment variables. There are therefore two `env.d` folders with different environment variables.
 
-Here is a list of the ones that are needed or optional:
+First, there is our Django backend. We try to follow [12 factors app](https://12factor.net/) and so use environment variables for configuration.
 
-## DJANGO_SETTINGS_MODULE
+Then, there is our aws deployment configurations in the `src/aws` folder. It looks in `src/aws/env.d`.
 
-Description
+## 1. Django backend environment
 
-:   Define the settings file to use
+Specified in `{REPO_ROOT}/env.d/development`.
 
-Type
+### General Django settings
 
-:   String
+#### DJANGO_CONFIGURATION
 
-Mandatory
+Define the configuration to use in settings.
 
-:   Yes
+- Type: `String`
+- Required: Yes
+- Default: `Development`
+- Choices: Currently only `Development` is available.
 
-Default
+#### DJANGO_DEBUG
 
-:   None
+Turns on/off debug mode.
 
-Choices
+- Type: Boolean
+- Required: No
+- Default: `True` if `DJANGO_CONFIGURATION` is set to `Development`, `False` otherwise
+- Choices: `True` or `False`
 
-:   Must be set to `marsha.settings`
+#### DJANGO_SECRET_KEY
 
-## DJANGO_CONFIGURATION
+Standard Django secret key used to make the instance unique and generate hashes such as CSRF tokens or auth keys.
 
-Description
+- Type: String
+- Required: Yes
+- Default: None
 
-:   Define the configuration to use in settings
+#### DJANGO_SETTINGS_MODULE
 
-Type
+Define the settings file to use, relative to `src/backend`.
 
-:   String
+- Type: `String`
+- Required: Yes
+- Default: `marsha.settings`
+- Choices: Must be set to `marsha.settings`.
 
-Mandatory
+### Marsha specific settings
 
-:   Yes
+#### DJANGO_BYPASS_LTI_VERIFICATION
 
-Default
+Whether to skip all LTI-related checks and accept all requests on LTI endpoints as valid. This is useful for development but should never be set to `True` on publicly accessible deployments.
 
-:   None
+Note: Can't be set to `True` unless `DJANGO_DEBUG` is `True` too.
 
-Choices
+- Type: Boolean
+- Required: No
+- Default: `False`
+- Choices: `True` or `False`
 
-:   Currently only `Development` is available
+#### DJANGO_JWT_SIGNING_KEY
 
-## DJANGO_SECRET_KEY
+Secret key used to sign JWTs. Those are used to communicate between the Django backend and authenticated third parties (including the frontend).
 
-Description
+- Type: string
+- Required: Yes
+- Default: None
```

Comment:
The library that is requiring this setting is setting it by default to the value of DJANGO_SECRET_KEY with the following explanation:

>  the SIGNING_KEY setting defaults to the value of the SECRET_KEY setting for your django project. Although this is the most reasonable default that Simple JWT can provide, it is recommended that developers change this setting to a value which is independent from the django project secret key. This will make changing the signing key used for tokens easier in the event that it is compromised.


I think we could do and explain the same WDYT?

Reply:
It seems to me that this is not in the scope of this PR? I have not completely understood it so maybe you could open an issue so we can address this.